src/styles/main.scss

```diff
@@ -213,6 +242,7 @@ $custom-classes: (
   sal-columns: (chat-message-footer, no-flex-grow),
   sal-button: (share-button, citation-button, feedback-button, feedback-up-icon, feedback-down-icon, feedback-up-icon-active, feedback-down-icon-active),
   sal-button-container: (dialog-button),
+  sal-image-container: (centered-image),
```

Comment:
@shu this part is needed so that `sal-image-container` is callable via `with sal.image("centered-image"):`